vita3k/app/src/app.cpp

```diff
@@ -69,7 +74,7 @@ void set_window_title(EmuEnvState &emuenv) {
         window_title,
         emuenv.current_app_title, emuenv.io.title_id,
         emuenv.cfg.backend_renderer,
-        emuenv.fps, emuenv.ms_per_frame,
+        emuenv.avg_fps, 1000 / (emuenv.avg_fps ? emuenv.avg_fps : 1),
```

Comment:
Revert, sorry, but keep fps

Reply:
because fps changes too fast it updated the window title extremelly fast and its impossible to read, i have to use average

Reply:
no have sense, revert this is very no sense, indows title display FPS, avg is no fps, is totaly diferent, revert it